_utils/compliance/validators/logo.py

```diff
@@ -0,0 +1,129 @@
+import argparse
+import os
+from functools import partial
+from pathlib import Path
+
+from PIL import Image
+
+from .base import Validator
+from .helpers import lighten_image, resize_canvas, square_canvas, transparent_background
+from .models import CheckError, CheckResult
+
+
+class LogoValidator(Validator):
+    @classmethod
+    def validate(cls, result: CheckResult, args: argparse.Namespace) -> None:
+        if not result.options.get("path"):
+            return
+
+        module_dir: Path = result.options["path"]
+        logo_path = module_dir / "logo.png"
+
+        check_logo_image(image_path=logo_path, result=result)
```

Comment:
Arf!! My bad!! The logo can be svg (see [AWS](https://github.com/SEKOIA-IO/automation-library/blob/main/AWS/logo.svg)).
Can you add support for svg file? Maybe just check the existence of the file. Thanks